rust/Cargo.toml

```diff
@@ -132,6 +136,9 @@ harness = false
 
 [profile.release]
 strip = true
-opt-level = "s"
```

Comment:
I wonder if we should still have this flag in python?

Reply:
Oh, lemme remove it from python side as well.